### ISSUE TITLE
config文件有bug

### ISSUE THREAD

**Issue by HXB-1997** [anshulpaigwar/Frustum-Pointpillars#10]
eval_input_reader: {
  record_file_path: "/home/anshul/es3cap/kitti_data/pp_data/kitti_val.tfrecord"
  class_names: ["Car"]
  batch_size: 2
  max_num_epochs : 160
  prefetch_size : 25
  max_number_of_voxels: 12000
  shuffle_points: false
  num_workers: 3
  anchor_area_threshold: 1
  remove_environment: false
  kitti_info_path: "/home/anshul/es3cap/kitti_data/pp_data/kitti_infos_test.pkl"//这里应该是kitti_infos_val.pkl
  kitti_root_path: "/home/anshul/es3cap/kitti_data"
}

仓库代码有问题，需要按照作者提供的readme修改
这个问题困扰我好几天。。。终于解决了
